low-level/files/opt/seagate/sspl/setup/resource_map/storage.py

```diff
@@ -280,6 +289,35 @@ def get_disk_groups_info(self):
                 })
         return dg_data
 
+    def get_drives_info(self):
+        """Update and return drives information in specific format."""
+        drive_data = []
+        drives = self.get_realstor_encl_data("drives")
+        for drive in drives:
+            slot = drive.get("slot", -1)
+            if slot == -1:
+                continue
+            uid = drive.get("durable-id")
+            status = drive.get("health", "NA")
+            description = drive.get("health", "NA")
+            recommendation = drive.get("health-recommendation", "NA")
+            specifics = [
+                {
+                    "serial-number": drive.get("serial-number", "NA"),
+                    "model": drive.get("model", "NA"),
+                    "size": drive.get("size", "NA"),
+                    "temperature": drive.get("temperature", "NA"),
+                    "disk-group": drive.get("disk-group", "NA"),
+                    "storage-pool-name": drive.get("storage-pool-name", "NA"),
+                    "location": drive.get("location", "NA")
```

Comment:
include this in health specifics 
                "enclosure-id":0,
                "drawer-id":0,
                "slot":0,
                "port":0,
                "scsi-id":0,
                "blocksize":512,
                "blocks":3907029168,
                "serial-number":"Z4H099ZE0000R6375N70",
                "vendor":"SEAGATE",
                "model":"ST2000NM0034",
                "revision":"E0B1",
                "architecture":"HDD",
                "interface":"SAS",
                "type":"SAS MDL",
                "locator-led":"Off",
                "enclosure-wwn":"500C0FF0A7117ABF",
                "virtual-disk-serial":"00c0ff3c638b0000a803955c00000000",
                "led-status":" Online",
                "power-on-hours":2847,

Also add logical volume info if available

Reply:
Done.
logical volume info is not available in disks response.